server/routes/comments.js

```diff
@@ -1,80 +1,95 @@
 var express = require ('express')
 var router = express.Router()
-const db = require ('../database/db.js')
+const comment = require('../database/queries/comment')
+
 
 //get all comments
 router.get('/', async (req, res) => {
     try {
-        let allComments = await db.any('SELECT * FROM comments;')
+        let allComments = await comment.getAllComments()
         res.json({
-            body: allComments
+            payload: allComments,
+            msg: 'success',
+            err: false
         })
     } catch (error) {
-        res.json({
-            error: error
+        console.log(error)
+        res.status(500).json({
+            payload: null,
+            msg: error,
+            err: true
         })
     }
 })
 
 //get comments by user id
-router.get('/:users_id', async (req, res) => {
-    try {
-        let commentByUserID = await db.any(`SELECT * FROM comments WHERE users_id = '${req.params.users_id}' `)
-        res.json({
-            payload: commentByUserID,
-            message:"got all comments by users id"
-        })
-    }catch (err){
-        console.log(err)
-        res.json({
-            error:err
-        })
-    }
-})
+// router.get('/:users_id', async (req, res) => {
+//     try {
+//         let commentByUserID = await db.any(`SELECT * FROM comments WHERE users_id = '${req.params.users_id}' `)
+//         res.json({
+//             payload: commentByUserID,
+//             message:"got all comments by users id"
+//         })
+//     }catch (err){
+//         console.log(err)
+//         res.json({
+//             error:err
+//         })
+//     }
+// })
 
     // get comments by video id 
     router.get('/video/:video_id', async (req, res) => {
+        const {video_id} = req.params
         try {
-            let commentByVideoID = await db.any(`SELECT * FROM comments WHERE video_id = '${req.params.video_id}' `)
+           const commentByVideoID = await comment.getCommentsByVideoID(video_id)
             res.json({
                 payload: commentByVideoID,
-                message:"got all comments by video id"
+                message:"got all comments by video id",
+                err: false
             })
         }catch (err){
-            console.log(err)
-            res.json({
-                error:err
+            console.log(error)
+            res.status(500).json({
+                payload: null,
+                msg: error,
+                err: true
             })
         }
     });
 
         //post a new comment
 router.post('/video/:video_id', async (req, res) => {
+    const {users_id, comment_body} = req.body
+    console.log(req.body)
```

Comment:
console logs are great for debugging but should generally not be committed  unless you are using them as a log for the deployed api.